### ISSUE TITLE
"Internal Server Error"

### ISSUE THREAD

**Issue by dneise** [fact-project/shifthelper_webinterface#22]
Internal server error is shown, when mysql server connection breaks down.
Seen by Daniela around x-mas and also today by us:
```
shifthelper_webinterface_1  | [2017-01-09 09:29:15,853] ERROR in app: Exception on /testCall [GET]
```

**Comment by dneise** [comment]
See also https://github.com/fact-project/shifthelper/issues/192

**Comment by maxnoe** [comment]
fixed in #24
